Classes/Base/Config.php

```diff
@@ -52,10 +52,12 @@ public static function get( array $paths, array $local = array() )
 			self::$config = $conf;
 		}
 
-		if( isset( $local['typo3']['tsconfig'] ) )
+		// Check for array
+		if( isset( $local['typo3']['tsconfig'] ) && is_array( $local['typo3']['tsconfig'] ) )
 		{
 			$tsconfig = \Aimeos\Aimeos\Base::parseTS( $local['typo3']['tsconfig'] );
-			$local = \TYPO3\CMS\Extbase\Utility\ArrayUtility::arrayMergeRecursiveOverrule( $local, $tsconfig );
+			// merge tsconfig to local, ignore empty values
+			$local = \TYPO3\CMS\Extbase\Utility\ArrayUtility::arrayMergeRecursiveOverrule( $local, $tsconfig, false, false );
```

Comment:
You're right! But I can revert this change because this is done at AbstractController now. Was my first try to prevent overridding but problem was not here. Think we can allow to override such thing by tsconfig, there the user has to know what he does. Will revert this...